test.js

```diff
@@ -12,10 +13,38 @@ test('async', async t => {
 	t.is(await locatePath(paths), 'index.js');
 	t.is(await locatePath(['nonexistant']), undefined);
 	t.is(await locatePath(['noop', 'unicorn'], {cwd: 'fixture'}), 'unicorn');
+	t.is(await locatePath(['index.js'], {type: 'directory'}), undefined);
+	t.is(await locatePath(['fixture'], {type: 'file'}), undefined);
+	t.is(await locatePath(['fixture']), 'fixture');
+
+	if (!isWindows()) {
```

Comment:
No need for a dependency for this. Just do `if (process.platform !== 'win32') {}`